doc/user/depends.xml

```diff
@@ -612,22 +605,33 @@ int main() { printf("Hello, world!\n"); }
 
       <para>
 
-      Note that ignoring some of the arguments
-      in your custom &Decider; function
-      is a perfectly normal thing to do,
-      if they don't impact the way you want to
-      decide if the dependency file has changed.
+      These attributes may not be present at the time of the
```

Comment:
This is not new content, it is moved up from lower to try to improve logical flow.